api/server/server.go

```diff
@@ -1557,3 +1564,11 @@ func ServeApi(job *engine.Job) error {
 
 	return nil
 }
+
+func toBool(str string) bool {
```

Comment:
I know this function is `Env.GetBool`, but god it's horrible... :-(

It's fine to keep it as it was before, but with all the recent PR to remove `engine.Job`, hasn't anyone already implemented that one somewhere else?